.golangci.example.yml

```
@@ -318,14 +318,17 @@ linters-settings:
     min-complexity: 10
 
   godot:
-    # comments to be checked: `declarations`, `toplevel`, or `all`
-    scope: declarations
+    # comments to be checked: `declarations`, `toplevel`, or `all` (default: declarations)
+    scope: toplevel
     # list of regexps for excluding particular comment lines from check
     exclude:
-      # example: exclude comments which contain numbers
-      # - '[0-9]+'
-    # check that each sentence starts with a capital letter
-    capital: false
+    # exclude todo and fixme comments
+    - "^fixme:"
+    - "^todo:"
+    # don't check that each sentence ends with a period (default: true)
```

```suggestion
    # check that each sentence ends with a period (default: true)
```

I think it's better to explain the meaning of the option instead of the meaning of value in the example.